src/web/middleware/error-handler-middleware.ts

```diff
@@ -41,6 +42,11 @@ export const errorHandlerMiddleware = (
 			res.status(HttpStatusCode.Forbidden).send({ message: err.message });
 			return next();
 		}
+
+		if (err instanceof PaidFigmaPlanRequiredUseCaseResultError) {
+			res.status(HttpStatusCode.PaymentRequired).send({ message: err.message });
```

Comment:
**note:** [HTTP 402 Payment Required](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/402) is a non-standard code but it is very convenient to communicate plan-related errors without introducing additional app-specific error codes.